plugins/woocommerce-blocks/assets/js/blocks/checkout/address-card/index.tsx

```diff
@@ -28,33 +26,28 @@ const AddressCard = ( {
 	target: string;
 	fieldConfig: FormFieldsConfig;
 } ): JSX.Element | null => {
-	const formattedCountry = isString( ALLOWED_COUNTRIES[ address.country ] )
-		? decodeEntities( ALLOWED_COUNTRIES[ address.country ] )
-		: '';
+	const addressFormats = getSetting< Record< string, string > >(
+		'addressFormats',
+		{}
+	);
```

Comment:
This should be part of each country instead of its top level setting.

Reply:
Done, moved `format` into each `countryData` if it has a format, and added `defaultAddressFormat` as a top level setting.